src/types/point.js

```diff
@@ -72,7 +73,7 @@ PointAnnotation.defaultRoutes = {
 };
 
 function pointInCircle(p, point) {
-	const {width} = point;
+	const width = point.width;
```

Comment:
+4 bytes to bundle size, lets revert this one too.

Reply:
OKI! Done! I'm gonna push in next minutes.
I didn't know if there is a standard approach for that. I thought `const {properties} = instance;` could be used when you have more than 1 property to get, and not based on amount of bytes.